BlockSettleHW/ledger/ledgerDevice.cpp

```diff
@@ -1168,8 +1169,7 @@ bool LedgerCommandThread::writeData(const QByteArray& input, const std::string&
 {
    logger_->error(logHeader + " - >>> " + input.toHex().toStdString());
    if (sendApdu(dongle_, input) < 0) {
-      logger_->error(
-         logHeader + " - Cannot write to device.");
+      logger_->error(fmt::format("{} - Cannot write to device.", logHeader));
```

Comment:
spdlog's logger already supports fmt::format internally, so no need to put fmt::format in its methods

Reply:
True, fixed. Thanks